src/lib/Preview.js

```diff
@@ -877,6 +891,10 @@ class Preview extends EventEmitter {
         // Start the preview duration timer when the user starts to perceive preview's load
         const previewDurationTag = Timer.createTag(this.file.id, DURATION_METRIC);
         Timer.start(previewDurationTag);
+
+        if (this.pageTracker) {
```

Comment:
The page tracking logic isn't really UI. Can we find a better home for our analytics setup/teardown?